docusaurus/docs/dev-docs/migration/v4-to-v5/breaking-changes.md

```diff
@@ -55,6 +55,7 @@ This page is part of the [Strapi v4 to v5 migration](/dev-docs/migration/v4-to-v
 
 - [Vite is the default bundler in Strapi 5](/dev-docs/migration/v4-to-v5/breaking-changes/vite)
 - [Strapi 5 uses react-router-dom v6](/dev-docs/migration/v4-to-v5/breaking-changes/react-router-dom-6)
+- [Webpack Aliases are removed in Strapi v5](/dev-docs/migration/v4-to-v5/breaking-changes/webpack-aliases-removed)
```

Comment:
```suggestion
- [Webpack Aliases are removed in Strapi 5](/dev-docs/migration/v4-to-v5/breaking-changes/webpack-aliases-removed)
```

I also tend to call it "v5", by habit. But the next version of Strapi should be called "Strapi 5" (removing the "v") 😄

Reply:
Updated thank you.